app/api_clients/control_panel_api.js

```diff
@@ -42,31 +42,39 @@ class DjangoError extends APIError {
   constructor(error) {
     super(error);
     this.original_message = this.message;
+    const pattern = /500 - "(.*)Request Method:/;
+    const match = this.original_message.match(pattern);
+    this.message = match[1]
+      .trim()
+      .replace(/\\n/g, '<br>\n');
```

Comment:
this is quite hairy parsing, can we have a test for it with example of input?

Reply:
Yeah, I'll write a test.